### PR TITLE
Did a fake thing

### PR DESCRIPTION
**Pull request by abbreviatedman**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const greeting = "Hello, Team!";
```
Comment (gregorytesto) [inline main.js:1]: This looks great. But you can also do this:
`let greeting = "Hello World"`

File: main.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+console.log(greeting);
```
Comment (gregorytesto) [inline main.js:2]: I love it!

**Review by gregorytesto — COMMENTED** [review]
Added comments to code

File: main.js (hunk 1 above)
Comment (abbreviatedman) [reply main.js:1]: nah im good

**Pull request closed without merging**